docs/user-guide/getting-started.md

```diff
@@ -11,175 +11,173 @@ The Ambassador Edge Stack is designed to run in Kubernetes for production. The m
-    ```
-
-2. Determine the IP address of your cluster by running the following command:
-
-    ```bash
-    kubectl get -n ambassador service ambassador -o 'go-template={{range .status.loadBalancer.ingress}}{{print .ip "\n"}}{{end}}'
-    ```
+1. Install the Ambassador Edge Stack
+2. Generate a domain name for you to access the Edge Policy Console and complete
+   advanced configuration
+3. Obtain a TLS certificate for that domain name
+4. Configure automatic TLS and HTTPS using that certificate
 
-    Your load balancer may take several minutes to provision your IP address. Repeat the provided command until you get an IP address.
+**To install the Ambassador Edge Stack:**
 
-    Note: If you are a **Minikube user**, Minikube does not natively support load balancers. Instead, use `minikube service list`. You should see something similar to the following:
+1. Download the `edgectl`file for your operating system:
 
-    ```bash
-    (⎈ |minikube:ambassador)$ minikube service list
-    |-------------|------------------|--------------------------------|
-    |  NAMESPACE  |       NAME       |              URL               |
-    |-------------|------------------|--------------------------------|
-    | ambassador  | ambassador       | http://192.168.64.2:31230      |
-    |             |                  | http://192.168.64.2:31042      |
-    | ambassador  | ambassador-admin | No node port                   |
-    | ambassador  | ambassador-redis | No node port                   |
-    | default     | kubernetes       | No node port                   |
-    | kube-system | kube-dns         | No node port                   |
-    |-------------|------------------|--------------------------------|
-    ```
+* [MacOS](https://metriton.datawire.io/downloads/darwin/edgectl)
+* [Linux](https://metriton.datawire.io/downloads/linux/edgectl)
+* [Windows](https://metriton.datawire.io/downloads/windows/edgectl.exe)
+* or use a [curl command](/reference/edgectl-download).
 
-    Use any of the URLs listed next to `ambassador` to access the Ambassador Edge Stack.
+2. Move the file into your PATH (for Windows users, move it into the Windows
+   Systems path).
+   * Print your PATH with `echo $PATH`
+3. Ensure the file is executable with the command `chmod a+x /usr/local/bin/edgectl`
+4. Run the executable file with the command `./edgectl`
+5. Now, run the following command: `edgectl install`
 
-3. Navigate to `http://<your-IP-address>` and click through the certificate warning for access to the Edge Policy Console interface. The certificate warning appears because, by default, the Ambassador Edge Stack uses a self-signed certificate for HTTPS.
-    * Chrome users should click **Advanced > Proceed to website**.
-    * Safari users should click **Show details > visit the website** and provide your password.
+    Your terminal will print something similar to the following:
```

Comment:
Instead of just pasting what the terminal will see, can we actually explain what this command is doing behind the scenes a bit?

Reply:
fixed